rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -494,6 +514,7 @@ def _handle_post_decoding(
                     transaction=transaction,
                     decoded_events=decoded_events,
                 )
+                break  # We only need to handle zap contracts once
```

Comment:
this is valid as you don't need to iterate over all the events for what is getting executed

Reply:
Do you ask for any change or is it just a comment?